src/main/java/pix/data/MyData.java

```diff
@@ -0,0 +1,136 @@
+package pix.data;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import pix.tasks.Deadline;
+import pix.tasks.Event;
+import pix.tasks.Task;
+import pix.tasks.ToDo;
+
+/**
+ * MyDate class to keep track of the list of tasks.
+ */
+public class MyData {
+    /** Arraylist to keep track of all tasks. */
+    private final ArrayList<Task> data = new ArrayList<Task>();

```

Comment:
Is it more appropriate to name this variable as tasks since it is an arraylist consisting of tasks which is the plural of task?